src/pybind/ceph_daemon.py

```diff
@@ -1,5 +1,5 @@
 # -*- mode:python -*-
```

Comment:
Do these files need any parameterization of python mode wrt tab behavior?

Reply:
I wasn't trying to mess around too much with python files since whitespace is imprtant there. And if the code is run through a pep8 check mixed tab/spaces are going to be flagged.